### PR TITLE
Spelling: Exceeds selected service period, –

### PR DESCRIPTION
**Pull request by comradekingu**
Not 100% sure about the service period string

### CODE REVIEW

File: weblate_web/templates/user.html [hunk 1]
```diff
@@ -23,7 +23,7 @@ <h1>{% trans "Your services" %}</h1>
                         {{ service.get_status_display }}
                         {% if not service.check_in_limits %}
                         <div class="alert error">
-                        {% trans "Exceeding service limits" %}
+                        {% trans "Exceeds selected service period" %}
```
Comment (nijel) [inline weblate_web/templates/user.html]: It's actually about limits - number strings/languages (as listed in https://weblate.org/hosting/)

Comment (comradekingu) [reply weblate_web/templates/user.html]: In my head "limitations" makes more sense, will think about it.

Comment (comradekingu) [inline weblate_web/templates/user.html]:
```suggestion
                        {% trans "Exceeds selected service offering" %}
```

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**